dask/dataframe/io/io.py

```diff
@@ -613,6 +617,98 @@ def from_delayed(
     return df
 
 
+@insert_meta_param_description
+def from_callable(
```

Comment:
Note that `from_callable` was added in this PR to facilitate testing/debugging.  It is not clear if this type of function belongs in the public API.